### PR TITLE
fix: make reset view available in sandwich mode context menu

### PR DESCRIPTION
**Pull request by dogfrogfog**
## Brief
- https://github.com/pyroscope-io/pyroscope/issues/1727

## Changes
- fix ability to reset flamegraph from ContextMenu components

https://user-images.githubusercontent.com/47758224/202425277-b60efba4-ee6a-4501-aa8d-8bfc72f5a63e.mov

## Concerns
-

### CODE REVIEW

File: packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.tsx [hunk 1]
```diff
@@ -265,11 +265,20 @@ export default function FlameGraphComponent(props: FlamegraphProps) {
         );
       };
 
+      const ResetItem = () => {
+        const isResetDisabled =
+          !(selectedItem.isJust && selectedItem.value) && !dirty;
```
Comment (eh-am) [inline packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.tsx:270]: why do we need to check `selectedItem.value` exists? if it's not a `Nothing` then it should have a value

Comment (dogfrogfog) [reply packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.tsx:270]: correct, i will fix it

File: packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphRenderer.tsx [hunk 2]
```diff
@@ -377,8 +376,9 @@ class FlameGraphRenderer extends Component<
   // so that the flamegraph doesn't rerender unnecessarily
   isDirty = () => {
     return (
+      !this.state.selectedItem.isNothing ||
```
Comment (eh-am) [inline packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphRenderer.tsx:379]:
```suggestion
      this.state.selectedItem.isJust ||
```

**Review by eh-am — APPROVED** [review]
👍 :)

**Pull request merged**